src/tests/functional/plugin/shared/src/behavior/compiled_model/device.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/compiled_model/device.hpp"
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+INSTANTIATE_TEST_SUITE_P(OVCompiledModel, OVCompiledModelCorrectDevice, ::testing::Values("CPU", "GPU", "MULTI:CPU(4),GPU(4)"));
+
+
+TEST_P(OVCompiledModelCorrectDevice, CompileModelWithCorrectDevice) {
+    ov::Core ie = createCoreWithTemplate();
+    OV_ASSERT_NO_THROW(auto compiled_model = ie.compile_model(actualNetwork, deviceName));
+}
+
+INSTANTIATE_TEST_SUITE_P(ov_compiled_model, OVCompiledModelIncorrectDevice, ::testing::Values("MULTI:CPU(4),GPU(4) ", "MULTI:CPU(4),GPU(4)a"));
```

Comment:
Why test instance is placed in shared_lib?

Reply:
Hello，this is my first time to contribute to openvino. I placed the test instance in shared_lib according to the issue suggestion.